app/src/main/res/layout/activity_main.xml

```diff
@@ -89,6 +90,90 @@
+                android:layout_gravity="center">
+                <LinearLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical"
+                    android:gravity="center">
+                    <TextView
+                        android:text="@string/adultLabel"
+                        android:layout_height="wrap_content"
+                        android:layout_width="wrap_content"
+                        style="@style/Text.Large"
+                        />
+
+                    <TextView
+                        android:id="@+id/adultCounter"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/one"
+                        style="@style/Text"/>
+                    <LinearLayout
+                        android:layout_width="100dp"
+                        android:layout_height="wrap_content">
+                        <Button
+                            android:id="@+id/adultDec"
+                            android:layout_width="0dp"
+                            android:layout_height="wrap_content"
+                            android:layout_weight="1"
+                            android:text="@string/minus"
+                            android:onClick="counterButtonTapped"/>
+                        <Button
+                            android:id="@+id/adultInc"
+                            android:layout_width="0dp"
+                            android:layout_height="wrap_content"
+                            android:layout_weight="1"
+                            android:text="@string/plus"
+                            android:onClick="counterButtonTapped"/>
+                    </LinearLayout>
+                </LinearLayout>
+
+                <LinearLayout
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical"
+                    android:gravity="center">
+                    <TextView
+                        android:text="@string/childLabel"
+                        android:layout_height="wrap_content"
+                        android:layout_width="wrap_content"
+                        style="@style/Text.Large"
+                        />
+
+                    <TextView
+                        android:id="@+id/childCounter"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/zero"
+                        style="@style/Text"/>
+                    <LinearLayout
+                        android:layout_width="100dp"
+                        android:layout_height="wrap_content">
+                        <Button
+                            android:id="@+id/childDec"
```

Comment:
you could potentially make a style that you can apply to all these buttons since you have 4 very similar buttons.